beamer/util.py

```diff
@@ -64,9 +64,12 @@ def transact(
     except (ContractLogicError, requests.exceptions.RequestException) as exc:
         raise TransactionFailed() from exc
 
-    return func.web3.eth.wait_for_transaction_receipt(
+    receipt = func.web3.eth.wait_for_transaction_receipt(
         txn_hash, timeout=timeout, poll_latency=poll_latency
     )
+    if receipt.status == 0:
+        raise TransactionFailed("Receipt status failed")
```

Comment:
TxReceipt has no such thing like error_message you can see example one that i created for test

Reply:
> TxReceipt has no such thing like error_message you can see example one that i created for test

Ah, that's unfortunate. Then I would just use something like `"unknown error"` instead of the existing message,
because the existing message may give an impression that getting the status or receipt failed.